src/DocumentationBrowserViewExtension/DocumentationBrowserView.xaml.cs

```diff
@@ -14,6 +20,10 @@ public partial class DocumentationBrowserView : UserControl, IDisposable
     {
         private const string ABOUT_BLANK_URI = "about:blank";
         private readonly DocumentationBrowserViewModel viewModel;
+        private const string VIRTUAL_FOLDER_MAPPING = "appassets";
+        static readonly string HTML_IMAGE_PATH_PREFIX = @"http://";
+
+        public string FallbackDirectoryName { get; set; }
```

Comment:
`internal`?

Reply:
@QilongTang changed to internal, check the next commit:
https://github.com/DynamoDS/Dynamo/commit/dc993a0e0a0f7f6354ebd14326178e725a95bae5